package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "query-string",
-  "version": "4.2.3",
+  "version": "4.4.0",
```

Comment:
Don't bump the version. Package maintainers want to control this themselves. This is also the wrong version. It would be 4.3.0.